docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+
+### Create a keystore user and add test fund
+
+Create a keystore user and store the credential on the target node (here, the node running on port 3650). Send a request to this API endpoint:
+
+```shell
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"keystore.createUser",
+    "params" :{
+        "username":"myUsername",
+        "password":"myPassword"
+    }
+}' -H 'Content-Type: application/json' 127.0.0.1:9650/ext/keystore
+```
+
+**Don't forget to replace "myUsername" and "myPassword" with your own.**
+
+> **Important**: You should only create a keystore user on a node that you operate, as the node operator has access to your plaintext password.
+
+Note the pre-funded "ewoq" private key, which is a test private key for getting some test AVAX in your account.
+
+```shell
+PrivateKey-ewoqjP7PxY4yr3iLTpLisriqt94hdyDFNgchSxGGztUrTXtNN
+```
+
+Import the private key to a C-chain address:
+
+```shell
+curl --location --request POST '127.0.0.1:9650/ext/bc/C/avax' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "method": "avax.importKey",
+    "params": {
+        "username":"username",
+        "password":"password",
+        "privateKey":"PrivateKey-ewoqjP7PxY4yr3iLTpLisriqt94hdyDFNgchSxGGztUrTXtNN"
+    },
+    "jsonrpc": "2.0",
+    "id": 1
+}'
+```
+
+Be thorough on the URL paths and the method namespace (again because of the subnets you might be running queries against the wrong one).
+
+Read more: [Funding a Local Network](https://docs.avax.network/build/tutorials/platform/fund-a-local-test-network)
+
+### Integrate with Metamask
+
+Set up Metamask to connect to a custom RPC address of the local network:
+
+#### Avalanche Mainnet Settings
+
+Network Name: Avalanche Mainnet C-Chain
+New RPC URL: https://api.avax.network/ext/bc/C/rpc
+ChainID: 43114
+Symbol: AVAX
+Explorer: https://snowtrace.io/
+
+#### FUJI Testnet Settings
+Network Name: Avalanche FUJI C-Chain
+New RPC URL: https://api.avax-test.network/ext/bc/C/rpc
+ChainID: 43113
+Symbol: AVAX
+Explorer: https://testnet.snowtrace.io/
+
+#### Local Testnet (Avalanche Local Simulator) Settings
+Network Name: Avalanche Local
+New RPC URL: http://localhost:9650/ext/bc/C/rpc
+ChainID: 43112
+Symbol: AVAX
+Explorer: N/A
+
+> **Local testnet**: Listening port might not be 9650 depending on if you run an example program in `avalanchego` node or `avalanche-simulator` run script. The latter is recommended for quick start and will have a listening node on port 9650.
+
+Create a new Metamask account by importing this private key `0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027` on Metamask
+
+![adding new account on metamask](https://docs.avax.network/assets/images/Metamask-Import-Account-17b4d3c6e167ebf8709ace5bc30001f6.png)
+
+If all went well, you should have a funded Metamask AVAX wallet for building app locally:
+
+![Metamask with AVAX funded](https://i.imgur.com/fkLXV17.png)
+
+### Optional
```

Comment:
why would I do this step? :)

Reply:
Maybe it belongs to `Create a Node project` session. It is confusing here. I would think it is optional for setting up your nodes

Reply:
Removed!